File: ios/Camera.m
//
//  Camera.m
//  pytorch_linking
//
//  Created by Leon Dai on 2020-05-20.
//

#import "React/RCTViewManager.h"
@interface RCT_EXTERN_MODULE(Camera, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(bestGuess, NSString)
RCT_EXPORT_VIEW_PROPERTY(confidence, float)
RCT_EXPORT_VIEW_PROPERTY(onStatusChange, RCTDirectEventBlock)
@end

File: ios/Camera.swift
//
//  Camera.swift
//  pytorch_linking
//
//  Created by Leon Dai on 2020-05-20.
//

import Foundation

@available(iOS 10.0, *)
@objc(Camera)

class Camera: RCTViewManager {
  override func view() -> UIView! {
    return CameraView()
  }
}
